src/components/CardItem/SmallCarditem.tsx

```diff
@@ -8,11 +8,18 @@ import { UiTypography } from '../ui';
 import { UiTooltip } from '../ui/UiTooltip';
 
 import { cardItemPossibilitiesStyles } from './styles';
+import styles from './styles.module.scss';
 
 function SmallCarditem({ item, imageList }: any) {
   return (
     <Stack sx={cardItemPossibilitiesStyles.wrapper}>
-      <Image src={item.imageSrc} alt="Card Image" width={80} height={80} />
+      <Image
```

Comment:
try this code

import Image from 'next/image';
import { Box } from '@mui/material';

const StyledImage = ({ src, alt, sx, ...rest }) => {
    return (
        <Box sx={sx}>
            <Image src={src} alt={alt} {...rest} />
        </Box>
    );
};

export default StyledImage;

and after delete scss file